app/scripts/services/constants.js

```diff
@@ -10,3 +10,57 @@ angular.module('confRegistrationWebApp').constant('ruleTypeConstants', {
   FORCE_SELECTION: 'FORCE_SELECTION',
   SHOW_OPTION: 'SHOW_OPTION',
 });
+
+export const states = [
```

Comment:
Is there a way to not hardcode this? Also aren't there other valid values like Puerto Rico and military things? 🤷‍♂️